packages/app/src/components/pages/Home/useSort.js

```diff
@@ -55,12 +55,14 @@ const useSort = () => {
     return sortableNoms
   }, [NominationsData, sortConfig])
 
-  const requestSort = (key) => {
-    let direction = 'ascending'
-
-    if (sortConfig && sortConfig.key === key && sortConfig.direction === 'ascending') {
+  const requestSort = (key, dontDoDefault) => {
```

Comment:
1. `dontDoDefault` is being used as a conditional, so we should use a boolean instead of an integer.
2. For boolean flags like this, it's better to avoid using negatives because it makes the logic confusing to read. `doDefault !== true` is easier to understand than `dontDoDefault !== false`.

Reply:
Update this functionality for code clarity.